resources/leiningen/new/xiana/test/app_name_test.clj

```diff
@@ -11,7 +11,7 @@
 (deftest index-test
   (is (= {:body   "Index page"
           :status 200}
-         (-> {:url                  "http://localhost:3000/"
+         (-> {:url                  "http://localhost:3333/"
               :unexceptional-status (constantly true)
               :method               :get}
              http/request
```

Comment:
I believe this test is just not working anymore, if the acl-restrict interceptor is executed.

Reply:
In order for the index page to be served, the `acl-restrict` interceptor is disabled (but exist as a comment for letting the user know how to use it). The test therefore works as expected.